src/components/widgets/EventCountdownWidget.tsx

```
@@ -0,0 +1,138 @@
+    if (isEmpty) {
+        return (
+            <a
+                href="/events"
+                className="block w-full max-w-sm mx-auto bg-gradient-to-br from-indigo-500 to-purple-600 rounded-3xl shadow-xl p-6 text-center text-white hover:scale-[1.02] transition-transform"
+            >
+                <div className="w-16 h-16 bg-white/20 rounded-full flex items-center justify-center mx-auto mb-4">
+                    <svg className="w-8 h-8 text-white" fill="none" stroke="currentColor" viewBox="0 0 24 24">
+                        <path strokeLinecap="round" strokeLinejoin="round" strokeWidth={2} d="M8 7V3m8 4V3m-9 8h10M5 21h14a2 2 0 002-2V7a2 2 0 00-2-2H5a2 2 0 00-2 2v12a2 2 0 002 2z" />
+                    </svg>
+                </div>
+                <h3 className="text-xl font-bold mb-2">Your Schedule is Clear</h3>
```

ESLint

[error] Replace `················` with `········`

(prettier/prettier)